.circleci/config.yml

```diff
@@ -40,14 +56,22 @@ jobs:
       steps:
           - checkout
           - run:
-                name: Change owner on project dir (default user = circleci (1001) and docker needs uid 1000)
+                name: Change ACLs on project dir (default user = circleci (1001) and docker needs uid 1000)
```

Comment:
I would fix directly the Dockerfile itself to modify uid and guid during build. This is cleaner in my opinion, because it allows to install the PIM on any laptop.

In dockerfile:

```
ARG USER_ID=1000
ARG USER_GROUP=1000

......
usermod --uid ${USER_ID} www-data && groupmod --gid ${USER_GROUP} www-data && \
```

And in the command, you can specify it with:

```
docker build --build-arg USER_ID=$(id -u) --build-arg USER_GROUP=$(id -g) ....
```

Already done successfully on some projects with Circle CI.

Reply:
Interesting suggestion, but not for this PR because the Dockerfile to modify is in CE and/or EE dev repositories